.eslintrc.js

```diff
@@ -211,15 +212,7 @@ module.exports = {
 				definedTags: [ 'jest-environment' ],
 			},
 		],
-		'react-compiler/react-compiler': [
-			'error',
-			{
-				environment: {
-					enableTreatRefLikeIdentifiersAsRefs: true,
-					validateRefAccessDuringRender: false,
-				},
-			},
-		],
+		'react-hooks/rules-of-hooks': 'warn',
```

Comment:
I think we also need to enable `react-compiler` manually:

https://github.com/facebook/react/blob/main/packages/eslint-plugin-react-hooks/src/index.ts#L16C4-L16C18

Haven't tested, but without that the react compiler rule just won't run, I think.